lib/snap-fingers.js

```diff
@@ -25,6 +25,24 @@ module.exports = () => {
 		logger.log(`😈 Snapping fingers inside '${cwd}' with full power...`);
 	}
 
+	const ironManWithGauntlet = readArgs(constants.arguments.ironManWithGauntlet);
+
+	if(ironManWithGauntlet) {
+		logger.log(`I... am... Iron Man!`);
+		const correctPaths = JSON.parse(fs.readFileSync(`${os.homedir()}/.to-dust/correct-paths.json`, 'utf8'));
```

Comment:
Hey, this will throw en error in case the `.to-dust` is not already existing.. You might want to first create the directory

Reply:
Sorry my bad, I put that checking condition below. I'll move it up.